zanata-war/src/test/resources/arquillian.xml

```diff
@@ -12,7 +12,7 @@
   <container qualifier="jbossas-managed">
     <configuration>
       <!-- We can use the JBOSS_HOME env variable instead of this -->
-      <property name="jbossHome">${arquillian.jboss.home}</property>
+      <property name="jbossHome">${jboss.home}</property>
```

Comment:
I remember @carlosmunoz is against this idea but can't remember exactly why. Carlos can you explain again?